revenue_app/templates/revenue_app/_sidebar.html

```diff
@@ -44,11 +44,11 @@
             </form>
         </a>
         <a href="#" class="list-group-item list-group-item-action flex-column cursor-default">
-            <form action={% url 'usd' %}  method="POST">
+            <form action="{% url 'usd' %}"  method="POST">
                 ARS-USD
-                <input type="number" step="0.01" min="0" class="form-control" name="usd_ars">
+                <input type="number" step="0.01" min="0.01" class="form-control" name="usd_ars">
```

Comment:
missing required attribute

Reply:
We can't put required attribute yet, because Cancel button will ask us to put something in that fields. We should validate this on the view.